sanket/javascript/day18/assignment4/sanketA4_Q2.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    no: <input type="text" id="no" />
+    <button onclick="cal()">calculate</button>
+    <p id="p"></p>
+
+    <script>
+      function cal() {
+        let n = parseInt(document.getElementById("no").value);
+        let r;
```

Comment:
change variable names don't use a, b, c, x, y, z as a variable name give proper relative name to the functionality 
and make this change in each and every assignment